internal/attestation/idkeydigest/idkeydigest.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: AGPL-3.0-only
+*/
+
+package idkeydigest
```

Comment:
Do you have Pro/Cons for this? My arguments why I created a new package were: 1. the digest is not a measurement and 2. if we choose to implement MAA in the future, we already have the place for this. Also idkeydigest is currently a leaf package and measurements is not, making importing more complex.

Reply:
See my comment below:
Makes it easier if we want to use automatic generation of this value.
However, its also possible to just define the default value in the `measurements` package as `[][]byte`, and then have another function in `idkeydigest` that converts it to the aliased type

Reply:
If its only about the interface with which you can get the default values then my suggestion would be to an indirection, where we don't export `MustGetDefaultIDKeyDigests()` rename and use it inside `idkeydigest.DefaultsFor(csp)`.

Reply:
Maybe instead of having the values as hex encoded strings that need to be parsed with potential panics, resulting in a global variable declaration in `config`, just having them as `[][]byte` inside the code would be enough.

Reply:
The idea for merging was mainly for automatic source code generation